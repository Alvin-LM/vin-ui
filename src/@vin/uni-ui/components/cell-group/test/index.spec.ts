import { mount } from '@vue/test-utils';
import CellGroup from '../index.vue';

test('should render title、desc slot correctly', () => {
  const wrapper = mount(CellGroup, {
    slots: {
      title: () => 'Custom Title',
      desc: () => 'Custom Desc',
    },
  });
  expect(wrapper.html()).toMatchSnapshot();
});
