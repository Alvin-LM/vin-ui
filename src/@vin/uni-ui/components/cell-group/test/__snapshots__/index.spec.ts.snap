// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render title„ÄÅdesc slot correctly 1`] = `
"<view class=\\"vin-cell-group\\">Custom TitleCustom Desc<view class=\\"vin-cell-group__warp\\"></view>
</view>"
`;
